hack/lib/util.sh

```diff
@@ -139,6 +135,23 @@ kube::util::gen-doc() {
   ${cmd} "${tmpdir}"
   # create the list of generated files
   ls "${tmpdir}" | LC_ALL=C sort > "${tmpdir}/.files_generated"
+
+  # remove all old generated file from the destination
+  for file in $(cat "${dest}/.files_generated" 2>/dev/null); do
+    if [[ -e "${tmpdir}/${file}" && ! -z "${skipline}" ]]; then
+      # If RCS format diff has 3 lines, only 1 line changed.
+      local fdiff=$(diff -n "${dest}/${file}" "${tmpdir}/${file}")
+      if [[ $(echo "${fdiff}" | wc -l) == 3 && $(echo "${fdiff}" | grep -e "${skipline}") ]]; then
```

Comment:
Changed skipline to prefix, and filtering file contents by `${prefix}` before comparing that contents are the same. PTAL